contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.5.0--4.0.0.sql

```diff
@@ -11,6 +11,26 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
  * final behaviour.
  */
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
+LANGUAGE C
+AS 'babelfishpg_common', 'babelfish_update_server_collation_name';
+
+SELECT sys.babelfish_update_server_collation_name();
+
+DROP FUNCTION sys.babelfish_update_server_collation_name();
```

Comment:
Do we really need this ? Original comment says that it is only needed in 2.3->3.x https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/contrib/babelfishpg_common/src/collation.c#L1585-L1586

Reply:
the comment says `This helper function would only be useful and strictly be used during 1.x->2.3 and 2.3->3.0 upgrade`.
I believe this comment was intended to say that this function execution should be strictly be used in all major version upgrade scripts, and as back then version 4.0 was not available its written only till 3.x.
cc: @Deepesh125

Reply:
yes, it should be applicable to 2.3 to all major version upgrade